### ISSUE TITLE
Theming issue with preview of multiple images

### ISSUE THREAD

**Issue by MGThePro** [CapnKitten/Material-Discord#175]
Discord seems to have a new feature where the image viewer works different when viewing an image from a message that has multiple images.

On the regular Discord, it just adds an arrow on the left and right to switch to the next image of that message, and clicking on the image itself will also close the image popup (where usually you'd have to click outside the image).

Most of that works with this theme, however the background is broken and turns most of the screen into a solid grey instead of just showing the discord window but darker.

So, basically, the steps to reproduce:

- upload multiple images in one message
- click on an image
- with material discord enabled, the background will be broken

This is happening on both Discord Stable 0.0.18 as well as PTB 0.0.30 and on the latest material discord 3.0.4

**Comment by MGThePro** [comment]
Strange, after a reboot it suddenly isn't happening anymore.
